File: bronze/2445/c++/main.cpp
#include <iostream>

int main()
{
    int num, temp;
    std::cin >> num;
    int i;
    for (i = 1; i <= num; i++) {
        temp = (num * 2) - (i * 2);
        for (int j = 0; j < i; j++) {
            std::cout << '*';
        }
        for (int j = 0; j < temp; j++) {
            std::cout << ' ';
        }
        for (int j = 0; j< i; j++) {
            std::cout << '*';
        }
        std::cout << '\n';
    }
    for (i = num -1; i > 0; i--) {
        temp = (num * 2) - (i * 2);
        for (int j = 0; j < i; j++) {
            std::cout << '*';
        }
        for (int j = 0; j < temp; j++) {
            std::cout << ' ';
        }
        for (int j = 0; j< i; j++) {
            std::cout << '*';
        }
        std::cout << '\n';
    }


    return 0;
}

File: bronze/2445/python/main.py

if (__name__ == "__main__"):
    num = int(input())

    for i in range(1, num +1):
        temp = (num * 2) - (i * 2)
        print('*' * i, end='')
        print(' ' * temp, end='')
        print('*' * i)
    i = num
    for _ in range(num):
        i -= 1
        temp = (num * 2) - (i * 2)
        print('*' * i, end='')
        print(' ' * temp, end='')
        print('*' * i)
